ui/imports/shared/popups/send/SendModal.qml

```diff
@@ -427,7 +428,7 @@ StatusDialog {
                         anchors.leftMargin: Style.current.bigPadding
                         anchors.rightMargin: Style.current.bigPadding
                         store: popup.store
-                        selectedAccount: store.selectedSenderAccount
+                        selectedAccount: popup.selectedSenderAccount
```

Comment:
wrong replacement here? The idea was to add the
property var preSelectedAccount: store.selectedSenderAccount
and replace all instances of  store.selectedSenderAccount with popup.preSelectedAccount